README.md

```diff
@@ -43,6 +43,13 @@ repos on GitHub, it is still a useful source of relative usage data.
   $ pip install -r requirements.txt
   ```
 
+  To update the `requirements.txt` file to the newest versions of dependencies, run:
+
+  ```sh
+  $ pip install -U pip-tools  # Make sure you have pip-compile installed
+  $ pip-compile --upgrade
+  ```
+
```

Comment:
Documentation is always necessary ;).

In the interest of automating things and making getting started for development less intimidating, the `pip-compile` step has been moved to the `Makefile`.